components/ShoppingListEditor/RecipeModalContent.js

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+
+import DeleteButton from "../buttons/DeleteButton";
+import IconPlusTextButton from "../buttons/IconPlusTextButton";
+import { GiCheckMark as CheckIcon } from "react-icons/gi";
+
+export default function RecipeModalContent({ recipe, onAdd, onDelete }) {
+  return (
+    <ModalWrapper>
+      <RecipeName>{recipe.name}</RecipeName>
+      {recipe.variant && (
+        <RecipeVariant>{`- ${recipe.variant} -`}</RecipeVariant>
+      )}
+      <RecipeItemsList>
+        {recipe.items.map(({ id, name, isOnList }) => (
+          <li key={id}>
+            <RecipeItemWrapper>
+              <RecipeItemName isOnList={isOnList}>{name}</RecipeItemName>
+              {!isOnList && (
+                <DeleteButton id={id} onDelete={() => onDelete(recipe, id)} />
```

Comment:
I am confused, a function, which seemingly calls it self. Or is it just the same naming for 2 different functions?

Reply:
`isOnList` is a value true or false and used to conditionally render if an item is already listed. I reach it through for the CSS rules. ;)